pymilvus/exceptions.py

```diff
@@ -213,3 +213,4 @@ class ExceptionsMessage:
     AmbiguousDeleteFilterParam = (
         "Ambiguous filter parameter, only one deletion condition can be specified."
     )
+    AmbiguousQueryFilterParam = "Ambiguous parameter, only ids or filter can be specified."
```

Comment:
Ambiguous parameter, either ids or filter should be specified, cannot support both.